docs/UserGuide.md

```diff
@@ -329,6 +344,7 @@ Action      | Command Format with Examples
 ------------|------------------
 **Add**     | `add i/MATRICULATION_NO n/STUDENT_NAME m/MODULE_CODE [p/PHONE_NUMBER] [t/TELEGRAM_HANDLE] [e/EMAIL_ADDRESS] ` <br> Example: `add i/AXXXXXXXR n/john m/CS2103T p/98765432 t/johnnn e/e0123456@u.nus.edu`
 **Delete**  | `delete STUDENT_INDEX...` (or) `delete i/STUDENT_ID` <br> Example: `delete 10` (or) `delete 10 20` (or) `delete i/AXXXXXXXR`
+**Delete Module**  | `delete m/MODULE_CODE` <br> Example: `delete m/CS2100`
```

Comment:
```suggestion
**Delete Module**  | `deleteModule m/MODULE_CODE` <br> Example: `delete m/CS2100`
```

Perhaps using `deleteModule` might be more consistent with the current `deleteTask` command that capitalises the "T"?